dashboards/php/php.json

```diff
@@ -114,18 +107,34 @@
 			  "thresholds": [
 				{
 				  "alertSeverity": "CRITICAL",
-				  "value": 1
+				  "value": 0
 				}
 			  ]
 			}
 		  },
 		  {
-			"title": "Success Rate",
+			"title": "",
+			"layout": {
+			  "column": 1,
+			  "row": 3,
+			  "width": 2,
+			  "height": 3
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.markdown"
+			},
+			"rawConfiguration": {
+			  "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/launcher/catalog-pack-details.launcher/?pane=eyJuZXJkbGV0SWQiOiJjYXRhbG9nLXBhY2stZGV0YWlscy5jYXRhbG9nLXBhY2stY29udGVudHMiLCJxdWlja3N0YXJ0SWQiOiI0NzVkZWM2OS0xMGM5LTRiYzYtODMxMi0zY2FhMjY2ZmIwMjgifQ==).\n\nDetect common [installation and configuration problems](https://download.newrelic.com/nrdiag/nrdiag_latest.zip) with New Relic agents. Optionally, attach this troubleshooting data to a support case. [Learn more](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics)\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=PHP&entry.358368110=https://onenr.io/0ERzlZYPvQr) here and let us know how we can improve it for you.\n"
```

Comment:
Need modification.
Here, "Detect common installation and configuration problems" is downloading a .zip folder, which is not the ideal scenario